mainwindow.ui

```diff
@@ -66,9 +66,6 @@
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
-      <property name="sizeAdjustPolicy">
```

Comment:
So we don't really loose anything feature wise and bug-wise by reintroducing Aleksey's workaround, except this property looks cleaner in the code.